azure-pipelines.deprecate-package.yml

```diff
@@ -15,26 +15,41 @@ parameters:
 
 variables:
   - group: 'Github and NPM secrets'
-  - template: .devops/templates/variables.yml
-    parameters:
-      skipComponentGovernanceDetection: false
   - name: tags
     value: production,externalfacing
 
-pool: '1ES-Host-Ubuntu'
+resources:
+  repositories:
+    - repository: 1esPipelines
+      type: git
+      name: 1ESPipelineTemplates/1ESPipelineTemplates
+      ref: refs/tags/release
 
-jobs:
-  - job: DeprecatePublishedPackage
+extends:
+  template: v1/1ES.Official.PipelineTemplate.yml@1esPipelines
+  parameters:
     pool:
-      name: '1ES-Host-Ubuntu'
-      image: '1ES-PT-Ubuntu-20.04'
-      os: linux
-    workspace:
-      clean: all
-    steps:
-      - template: .devops/templates/tools.yml@self
-
-      - script: |
-          npm deprecate ${{ parameters.packageSpec }} "${{ parameters.message }}" --registry https://registry.npmjs.org/ --//registry.npmjs.org/:_authToken=$(npmToken)
-        displayName: 'Deprecate package'
-        condition: eq(variables.dryRun, false)
+      name: Azure-Pipelines-1ESPT-ExDShared
+      image: windows-latest
+      os: windows # We need windows because compliance task only run on windows.
+    stages:
+      - stage: main
+        jobs:
+          - job: Release
```

Comment:
is this naming enforced by 1ES , if not can we tweak it ( "Deprecate Release" for eg)  ?

Reply:
Updated job name to `DeprecateRelease`